pandas/core/frame.py

```diff
@@ -2091,10 +2091,18 @@ def to_feather(self, path) -> None:
     @Substitution(klass="DataFrame")
     @Appender(_shared_docs["to_markdown"])
     def to_markdown(
-        self, buf: Optional[IO[str]] = None, mode: Optional[str] = None, **kwargs
+        self,
+        buf: Optional[IO[str]] = None,
+        mode: Optional[str] = None,
+        index: Optional[bool] = None,
```

Comment:
default is True, no?

Reply:
default=true means disable setindex to user. but the default behaviour is still with index tho